my_filt.py

```diff
@@ -0,0 +1,16 @@
+def my_filter(func, lst):
+    """
+    function implementing filter using tail recursion
```

Comment:
Isnt there a more simple way to implement this function?
Why using recursion? Is it more efficient?

Reply:
Explain what is filter.